src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -72,7 +87,7 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
         {
             if ( from == null || to == null || actual == null)
                 throw new ArgumentException( "Cannot compare using a null reference", "actual" );
```

Comment:
The `IComparer` interface is designed to work with null references, so I strongly expect it to work here. I'm positive I will hit this in real world code.